plugins/lime-plugin-align/src/alignEpics.js

```diff
@@ -73,8 +71,8 @@ const initAlign = (action$, state$ ) =>
 		})
 		.mergeMap((res) => Observable.from([
 			({ type: STATION_SET, payload: res }),
-			({ type: IFACE_SET, payload: res? res.iface: undefined }),
-			({ type: TIMER_START })]));
+			({ type: IFACE_SET, payload: res? res.iface: undefined })
+		]));			//({ type: TIMER_START })]));
```

Comment:
Is there still use for this? I'd say either remove or add a comment.

Reply:
can be deleted